setup_env.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+clear
```

Comment:
It's considered "rude" for a script clear the terminal scrollback buffer like that.

```suggestion
```

Reply:
100% agree, i just "converted" the batch files which does cls, should i remove them from the batch files too?

Reply:
I'd say so, but @DevilXD might think otherwise. :p

Reply:
I'd say so, but @DevilXD might think otherwise. :p